bin/runtime/src/lib.rs

```diff
@@ -143,11 +143,26 @@ parameter_types! {
     pub const SS58Prefix: u8 = ADDRESSES_ENCODING;
 }
 
+pub enum CallFilter {}
```

Comment:
nit:
```suggestion
pub enum CallFilter;
```

Reply:
this is invalid syntax for enum (would work for struct though)

Reply:
oh, why is this enum >;

Reply:
https://github.com/Cardinal-Cryptography/aleph-node/pull/1596#discussion_r1470307989 :P